README.md

```diff
@@ -241,7 +243,7 @@ bf2a09e erm, not sure why CI has broken so another go
 067c88e gah, I'm stupid. I can see why CI broke
 ```
 
-This is not linear because you're fixing something inside the PR. This should be [rebased](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)
+This is not linear because you're fixing something inside the PR. This should be [rebased](https://github.com/Planning-Inspectorate/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)
```

Comment:
did you just move this to confluence?

Reply:
I copied it (for now). Probably should update the link.